vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -157,11 +157,11 @@ void Unblinded::OnUnblindedTokens(
       continue;
     }
 
-    if (item->value + current_amount > reconcile.fee) {
+    if (current_amount >= reconcile.fee) {
       break;
     }
 
-    current_amount += item->value;
+    current_amount += 0.25;
```

Comment:
we need to hard code it in version 1.3.x as we can't do db migration. This was done through db in version 1.4.x